courts_db/data/courts.json

```diff
@@ -15646,7 +15710,9 @@
         "regex": [
             "Massachusetts {super}",
             "State of Massachusetts Superior Court",
-            "Superior Court of Massachusetts"
+            "Superior Court of Massachusetts",
+            "Commonwealth of Massachusetts Superior Court",
+            "Commonwealth of Massachusetts Superior Court,? (MIDDLESEX|Middlesex)"
```

Comment:
Do we actually need both of these regex patterns here for middlesex.  I assume we would have some normalization of capitalization

Reply:
Also - I assume that we have a supreior court breakdown here.